podman/api_connection.py

```diff
@@ -13,20 +13,26 @@
 
 
 class ApiConnection(HTTPConnection, AbstractContextManager):
-    """ ApiConnection provides a specialized HTTPConnection
-        to a Podman service."""
-
-    def __init__(self, url, base="/v1.24/libpod", *args,
-                 **kwargs):  # pylint: disable-msg=W1113
+    """
+    ApiConnection provides a specialized HTTPConnection
+    to a Podman service.
+    """
+
+    def __init__(
+        self, url, base="/v1.24/libpod", *args, **kwargs
```

Comment:
I don't think we should format function function parameters like this.  Is there a specific rule this is following?  The previous method was more in line with the pep8 rules.

Reply:
@mwhahaha I'm fine with letting the `black` formatter drive the style.  It is very opinionated and is almost PEP8.

Reply:
Works for me if that is preferred. Just need to document it.